test/blueprint/sbs-blueprint.spec.js

```diff
@@ -0,0 +1,65 @@
+const path = require('path');
+const assert = require('yeoman-assert');
+const helpers = require('yeoman-test');
+const Generator = require('yeoman-generator');
+const expectedFiles = require('../utils/expected-files');
+const constants = require('../../generators/generator-constants');
+
+const ANGULAR = constants.SUPPORTED_CLIENT_FRAMEWORKS.ANGULAR;
+
+const mockBlueprintSubGen = class extends Generator {
+    constructor(args, opts) {
+        super(args, opts);
+        const jhContext = (this.jhipsterContext = this.options.jhipsterContext);
+        if (!jhContext) {
+            this.error("This is a JHipster blueprint and should be used only like 'jhipster --blueprint myblueprint')}");
```

Comment:
should be blueprints (with 's')

Reply:
Copy/paste problem.
Updated every test description with same error.